src/Microsoft.ML.Data/Commands/ScoreCommand.cs

```diff
@@ -247,9 +241,8 @@ public static IDataScorerTransform GetScorer(IPredictor predictor, RoleMappedDat
             env.CheckValueOrNull(customColumns);
             env.CheckValueOrNull(trainSchema);
 
-            var schema = TrainUtils.CreateRoleMappedSchemaOpt(input.Schema, featureColName, groupColName, customColumns);
-            ISchemaBoundMapper mapper;
-            var sc = GetScorerComponentAndMapper(predictor, scorer, schema, env, out mapper);
+            var schema = new RoleMappedSchema(input.Schema, label: null, feature: featureColName, group: groupColName, opt: true);
```

Comment:
I think you lost customColumns here.
at least we used to pass them.

Reply:
Whoops

---
In reply to: [200457252](https://github.com/dotnet/machinelearning/pull/496#discussion_r200457252) [](ancestors = 200457252)